src/app/core/services/ess-info-error-message.service.ts

```diff
@@ -0,0 +1,38 @@
+import { Injectable } from '@angular/core';
+import { BehaviorSubject } from 'rxjs';
+
+export interface RequestedProductsNotInExchangeSet{
+  productName: string;
+  reason: string;
+}
+export interface InfoErrorMessage {
+  showInfoErrorMessage: boolean;
+  messageType: 'info' | 'warning' | 'success' | 'error';
+  messageDesc: string;
+  messageTitle?: string;
+  requestedProductsNotInExchangeSet?: RequestedProductsNotInExchangeSet[];
```

Comment:
As discussed, kindly check whether we need this property or not.

Reply:
requestedProductsNotInExchangeSet removed